<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceOptions</class>
 <widget class="QDialog" name="FindReplaceOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find / Replace Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbLookIn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Loo&amp;k in:</string>
       </property>
       <property name="buddy">
        <cstring>lookWhere</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lookWhere">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QCheckBox" name="matchWholeWord">
          <property name="text">
           <string>Match whole word onl&amp;y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="matchCase">
          <property name="text">
           <string>Matc&amp;h case</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="matchMinimal">
          <property name="toolTip">
           <string>&lt;p&gt;Make the expression match as little as possible.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Minimal ma&amp;tching</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Search mode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="searchNormal">
          <property name="toolTip">
           <string>&lt;p&gt;Normal, character by character comparison.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Nor&amp;mal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="searchWildcard">
          <property name="toolTip">
           <string>&lt;p&gt;Wildcard matching.&lt;/p&gt;
&lt;p&gt;&lt;b&gt;?&lt;/b&gt; - Matches any single character.&lt;p&gt;
&lt;p&gt;&lt;b&gt;*&lt;/b&gt; - Matches zero or more of any characters.&lt;p&gt;
&lt;p&gt;&lt;b&gt;[...]&lt;/b&gt; - Character set.&lt;p&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Wildcard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="searchRegex">
          <property name="toolTip">
           <string>&lt;p&gt;Perl-like regular expressions, with minor differences.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Re&amp;gular expression</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindReplaceOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindReplaceOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
